src/page-modules/assistant/layout.tsx

```diff
@@ -53,6 +59,15 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
     getTransportModeFilter,
   );
 
+  useEffect(() => {
+    if (orgId === 'nfk') {
+      tripQuery.transportModeFilter =
```

Comment:
Directly modifying the tripQuery object won't update the URL, will it? The rendering of the filter will work I guess, since `filterState={tripQuery.transportModeFilter}` is passed as a prop. 

I guess the URL is set correctly when the first search is performed.

> [!IMPORTANT]
> If the transport mode filter is set already, we should not overwrite it and change it. E.g. the URL is copied from somewhere with a set transport mode filter (which may intentionally include air transport), that filter should be untouched.